scripts/run-test-suite

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env bash
 
-set -e
```

Comment:
I'd feel it might be safer to catch any errors here and code the script to handle the exit codes that we care about. WDYT?

Reply:
I agree that it's probably safer. Mind updating this pull request with a proposed change?

Reply:
I think I've got it figured out. I didn't know that `set -e` ignores the conditions of `if` statements.